### DISCUSSION TITLE
upgrade to version alpha.41 error

### DISCUSSION THREAD

**Discussion by shallingzhang** [vuejs/vitepress#1859] [Q&A]
![image](https://user-images.githubusercontent.com/105492325/215310654-64161c4a-5c1d-4d10-8c62-675b43f9ae1d.png)
/vitepress-starter/node_modules/shiki/themes/material-theme-palenight.json ,the file is renamed material-palenight.json

**Comment by brc-dd** [comment]
Delete your lock file and node_modules, then reinstall latest version of vitepress it should work.

**Reply by shallingzhang** [reply]
i have deleted lock file and node_modoules , reinstall  latest version will happen same problem, it does not work also ,there is no generated material-palenight.json file ,but  i  copy material-theme-palenight.json to renamed material-palenight.json in the same path ,it will run normal.

**Reply by brc-dd** [reply]
In your config, is there any reference to `material-palenight`? Because we don't have it. The default theme is defined here: 

https://github.com/vuejs/vitepress/blob/010b3e5ad99f5e61fd01e27d0c3144896a8f3d86/src/node/markdown/plugins/highlight.ts#L59

and `material-theme-palenight.json` is already in your node_modules/shiki/themes. So, unless you have manually passed it through config.markdown.theme, it should not have searched for `material-palenight.json`.
